docker/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM python:3
```

Comment:
This DockerFile is not for OMAM GitHub Users, rather to docker hub auto build settings to use as repo source. Why do you want this to be added into OMAM GitHub repository? Can't we use bitbucket?

Reply:
@rajeevarakkal Having dockerfile along with OMAM github repository enables customers with isolated environment(no access to docker hub) to create image on their docker host instead of pulling it from docker cloud.

Reply:
@saishivarcr - please add all use cases into the above discussion points,  my review is based on the above points which we have discussed along with Technologist. If you keep on add/derive use cases it will be difficult for  review. **Acknowledge once you update all the use cases and objectives.**

However, How your script will pull python 3 image without dockerhub access ? Also do you expect a customer with GitHub access allowed and no docker hub access? Any specific reasons? is docker hab any know vulnerabilities?

Reply:
@rajeevarakkal Customers may have private registry configured in their environment, as python:3 is widely used there are good chances that it is already pushed onto the private registry. And as always it's good practice to expose Dockerfile so that customer gets confidence with the kind of image he is downloading.

Reply:
Right, Lets deliver this script as a `sample` and let customer use base image of their choice. We no need to enforce customer to use a particular open source os image we ship (which we are not sure about the secure configurations and presence of any malicious binaries in it) rather let them use their on IT certified secure images.